base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -0,0 +1,147 @@
+
+    @api.model
+    def apply_inheritance_specs(self, source, specs_tree, inherit_id):
+        for specs, handled_by in self._iter_inheritance_specs(specs_tree):
+            source = handled_by(source, specs, inherit_id)
+        return source
+
+    @api.model
+    def _iter_inheritance_specs(self, spec):
+        if spec.tag == "data":
+            for child in spec:
+                for node, handler in self._iter_inheritance_specs(child):
+                    yield node, handler
+            return
+        if spec.get("position") == "attributes":
+            if all(not c.get("operation") for c in spec):
+                yield spec, self._get_inheritance_handler(spec)
+                return
+            for child in spec:
+                node = etree.Element(spec.tag, **spec.attrib)
+                node.insert(0, child)
+                yield node, self._get_inheritance_handler_attributes(child)
+            return
+        yield spec, self._get_inheritance_handler(spec)
+
+    @api.model
+    def _get_inheritance_handler(self, node):
+        handler = super(IrUiView, self).apply_inheritance_specs
+        if hasattr(self, "inheritance_handler_%s" % node.tag):
+            handler = getattr(self, "inheritance_handler_%s" % node.tag)
+        return handler
+
+    @api.model
+    def _get_inheritance_handler_attributes(self, node):
+        handler = super(IrUiView, self).apply_inheritance_specs
+        if hasattr(self, "inheritance_handler_attributes_%s" % node.get("operation")):
+            handler = getattr(
+                self, "inheritance_handler_attributes_%s" % node.get("operation")
+            )
+        return handler
+
+    @api.model
+    def _list_variables(self, str_dict):
+        """
+        Store non literal dictionary values into a list to post-process
+        operations.
+        """
+        variables = []
+        items = str_dict.replace("{", "").replace("}", "").split(",")
+        for item in items:
+            key_value = item.split(":")
+            if len(key_value) == 2:
+                value = key_value[1]
+                if not ("'" in value or '"' in value):
+                    variables.append(value.strip())
+        return variables
+
+    @api.model
+    def inheritance_handler_attributes_python_dict(self, source, specs, inherit_id):
+        """Implement
+        <$node position="attributes">
+            <attribute name="$attribute" operation="python_dict" key="$key">
+                $keyvalue
+            </attribute>
+        </$node>"""
+        node = self.locate_node(source, specs)
+        for attribute_node in specs:
+            str_dict = node.get(attribute_node.get("name")) or "{}"
+            variables = self._list_variables(str_dict)
+            my_dict = safe_load(str_dict)
+            my_dict[attribute_node.get("key")] = attribute_node.text
+            for k, v in my_dict.items():
+                my_dict[k] = UnquoteObject(v) if v in variables else v
```

Comment:
```suggestion
                my_dict[k] = UnquoteObject(v) if v in variables else v
            my_dict[attribute_node.get("key")] = UnquoteObject(attribute_node.text)
```

The `attribute_node.text` needs to be unquoted in order to make `account_cost_center` module work.

Reply:
Why??... How do you use the inheritante? Can you give me it to see it?

Reply:
Sorry... I see the problem